src/services/append-to-gitignore-service.ts

```diff
@@ -3,16 +3,16 @@ import * as fs from 'fs'
 class AppendToGitignoreService {
   run() {
     const file = '.gitignore'
-    const addedForDotenvMsg = '# Ignore (and don\'t ignore) for dotenv.'
 
     // 1. create .gitignore if doesn't exist
     if (!fs.existsSync(file)) {
       fs.writeFileSync(file, '')
     }
 
-    if (fs.readFileSync(file).indexOf(addedForDotenvMsg) === -1) {
+    const fileContent = fs.readFileSync(file)
+    if (fileContent.indexOf('.env') === -1 || fileContent.indexOf('.env.me') === -1 || fileContent.indexOf('!.env.project') === -1) {
```

Comment:
this should not use `||` but I am going to go ahead and merge and then modify. 

I see where you are going with this and it makes sense. Thank you for raising the issue with some code.